frontend_app_admin_workspace_user/src/css/index.styl

```diff
@@ -145,6 +145,11 @@
       width 15%
     &__administrator
       width 15%
+    &__line
+      &__username
+        unicode-bidi plaintext
+      &__email
+        unicode-bidi plaintext
```

Comment:
I think the e-mails should be left-align even if they start with arabic characters